app/models/subject.rb

```diff
@@ -3,4 +3,71 @@ class Subject < ApplicationRecord
   has_many :courses, through: :course_subjects
 
   validates :code, uniqueness: true
+
+  SUBJECTS = %i[
```

Comment:
wondering if this should be renamed and/or live elsewhere as this is the reports mapped subject groups (+ the subtotal row names)

Reply:
Elsewhere we've set constants in initializers, eg https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/initializers/ethnicities.rb

Reply:
Named sth like `MinisterialITTReport::SUBJECTS` and `MinisterialITTReport::STEM_SUBJECTS` etc